caravan.lua

```diff
@@ -95,6 +97,45 @@ function commands.leave(...)
     end
 end
 
+local function isDisconnectedPackAnimal(unit)
+    if unit.following then
+        local dragger = unit.following
+        return (
+            unit.relationship_ids[ df.unit_relationship_type.Dragger ] == -1 and
+            dragger.relationship_ids[ df.unit_relationship_type.Draggee ] == -1
+        )
+    end
+end
+
+local function getPrintableUnitName(unit)
+    local visible_name = dfhack.units.getVisibleName(unit)
+    local profession_name = dfhack.units.getProfessionName(unit)
+    if visible_name.has_name then
+        return dfhack.TranslateName( visible_name ) .. ' (' .. profession_name .. ')'
+    else
+        return profession_name  -- for unnamed animals
+    end
```

Comment:
```suggestion
    if visible_name.has_name then
        return ('%s (%s)'):format(dfhack.TranslateName(visible_name), profession_name)
    end
    return profession_name  -- for unnamed animals
```

Reply:
I don't like this change as it would print " (Horse)", while I want to print units as they appear in unit list.

Reply:
I will test it and commit.